tests/pkg/module/Module/Basic.lean

```diff
@@ -64,6 +64,9 @@ has type
   Vector Unit 1
 but is expected to have type
   Vector Unit f
+
+Note: The following definitions were attempted to be unfolded but they are not marked as `@[expose]`:
```

Comment:
Personal opinion/bikeshed: I don't love how this message subtly suggests that adding the expose attribute is a good idea. I would prefer the more factual
```suggestion
Note: The following definitions were not unfolded because they are not marked as `@[expose]`:
```
or
```suggestion
Note: The following definitions were not unfolded because their definition is not exposed:
```

Reply:
Yeah, there is a balance here between not being helpful and being too helpful. I think the latter suggestion should work.